Assets/MRTK/Core/Utilities/Gltf/Schema/GltfAccessorSparseIndices.cs

```diff
@@ -30,6 +31,26 @@ public class GltfAccessorSparseIndices : GltfProperty
         /// `5123` (UNSIGNED_SHORT)
         /// `5125` (UNSIGNED_INT)
         /// </summary>
-        public GltfComponentType ComponentType;
+        public GltfComponentType ComponentType { get; set; }
+
+        [SerializeField]
+        private string componentType = null;
+
+        void ISerializationCallbackReceiver.OnAfterDeserialize()
+        {
+            if (Enum.TryParse(componentType, out GltfComponentType result))
+            {
+                ComponentType = result;
+            }
+            else
+            {
+                ComponentType = default;
```

Comment:
Does this work in the UWP build? For some reason I feel like I remember `default` vs `default(GltfComponentType)` isn't allowed in the versions of things Unity uses

Reply:
I have confirmed that it indeed works on HL2.